packages/gdpr-tool/package.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "@commercetools/gdpr-tool",
```

Comment:
Nitpick: either `gdpr-tools` or  simply `gdpr`.

In case we want to make this a CLI, I would suggest naming this `gdpr-cli`

Reply:
Good idea. Will wrap it into CLI 👍